gamemodes/terrortown/entities/weapons/weapon_tttbasegrenade.lua

```diff
@@ -84,6 +85,107 @@ function SWEP:SecondaryAttack()
 
 end
 
+---
+-- @param Vector P The initial position for this iteration cycle.
+-- @param number V The launch velocity for this iteration cycle.
+-- @param number G The gravity multiplier for this iteration cycle.
+-- @param number T The step size for the current iteration cycle.
+-- @return Vector The next location to iterate from.
+local function PositionFromPhysicsParams(P, V, G, T)
+	local A = G * physenv.GetGravity()
+
+	return P + (V * T + 0.5 * A * T ^ 2)
+end
+
+if CLIENT then
+	---
+	-- @realm client
+	local trajectory_ui = CreateConVar("ttt2_grenade_trajectory_ui", 0, FCVAR_ARCHIVE)
```

Comment:
to be in style with our other convar variables, this should probably be `cvEnableTrajectoryUI`